net-mgmt/pfSense-pkg-nrpe/files/usr/local/pkg/nrpe.inc

```diff
@@ -270,14 +270,13 @@ function nrpe_custom_php_validation_command($post, &$input_errors) {
 
 function nrpe_sync_config() {
 	global $config;
-	if (!is_array($config['hasync'])) {
+	$system_carp = $config['hasync'];
+	if (!is_array($system_carp) ||
```

Comment:
You should use code more like this, to avoid potential PHP quirks:

```php
init_config_arr(array('hasync'));
$system_carp = $config['hasync'];
if (!empty($system_carp) ||
```

(Applies to all of these)

Reply:
Done